### PR TITLE
Allow disabling colors

### PR DESCRIPTION
**Pull request by kodie**
Just a tiny implementation that allows us to disable colors:

```php
$command = new Ahc\Cli\Input\Command('test', 'My Test Command');

$command
  ->option('-n --no-colors', 'Disables terminal colors')
  ->parse(['thisfile.php', '--no-colors']);

if (!$this->colors) {
  Ahc\Cli\Output\Color::$enabled = false;
}
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -508,6 +508,12 @@ Ahc\Cli\Output\Color::style('error', [
 ]);
 ```
 
+#### Disable colors
+
+```php
+Ahc\Cli\Output\Color::$enabled = false;
```
Comment (adhocore) [inline README.md:514]: in addition to manual control like this, there can be env vars like LS_COLOR/S, NO_COLOR/S that you may want to check together while at it :)

Comment (kodie) [reply README.md:514]: LS_COLOR seemed to be specifically for the `ls` command but I like the idea of checking for NO_COLOR as described here: https://no-color.org/

So it'll now disable colors automatically of NO_COLOR environment variable is set to truthy.

**Comment by adhocore** [comment]
how does it play with html like color of this pkg? eg `<boldRed>text</end><eol>` should output just "text\n"

**Comment by adhocore** [comment]
cool, thanks 👍

**Pull request merged**